codalab/lib/bundle_cli.py

```diff
@@ -1054,6 +1054,11 @@ def do_store_command(self, args):
                 "url": args.url,
                 "authentication": args.authentication,
             }
+            if args.storage_type is None:
```

Comment:
Great! Can you update the cli tests so they include one example of not passing in the --storage-type argument?